schedule/src/main/java/itcast/mail/application/MailService.java

```diff
@@ -0,0 +1,26 @@
+package itcast.mail.application;
+
+import com.amazonaws.AmazonClientException;
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailService;
+import itcast.mail.dto.request.EmailSender;
+import itcast.mail.dto.request.SendMailRequest;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class MailService {
+
+    private final AmazonSimpleEmailService amazonSimpleEmailService;
+    private final EmailSender emailSender;
+
+    public void send(final SendMailRequest sendMailRequest) {
+        try {
+            amazonSimpleEmailService.sendEmail(emailSender.from(sendMailRequest));
```

Comment:
전 이렇게 매개변수로 바로 집어넣어서 합쳐놓는 것보다 분리해서 중간 변수로 명시적으로 작성하는 것이 더 가독성이 좋다고 생각하는데, 아래처럼 코드 두줄로 나타내는 것은 어떨까요?! + 변수를 아래보다 좀 더 명확히 적어주면 더 좋을거같아요~!
```suggestion
            SendEmailRequest emailRequest = emailSender.from(sendMailRequest);
            amazonSimpleEmailService.sendEmail(emailRequest);
```